frontend/spree_frontend.gemspec

```
@@ -25,6 +25,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'bootstrap',       '~> 4.3.1'
   s.add_dependency 'glyphicons',      '~> 1.0.2'
   s.add_dependency 'canonical-rails', '~> 0.2.5'
+  s.add_dependency 'inline_svg',      '~> 1.5.2'
```

@pmilewski let's loosen up this dependency
```suggestion
  s.add_dependency 'inline_svg',      '~> 1.5'
```